### ISSUE TITLE
Files moved out of unlocked encrypted directory cannot be accessed (Required key not available)

### ISSUE THREAD

**Issue by richardjharris** [google/fscrypt#419]
I am not exactly sure what happened. I think I drag and dropped a directory within an fscrypt-encrypted, unlocked directory, to another place. The files have their standard plaintext names and were moved from an unlocked directory. When I try to read or write to them, I get 'Required key not available'.

Kernel is 6.12.4

I still have all the protectors that were used loaded, created one directory per protector and unlocked it, but the files still cannot be accessed.

**Comment by ebiggers** [comment]
Yes, it is possible to end up with encrypted files outside an encrypted directory this way.  (It is not recommended to do this.)  They remain encrypted by the same encryption policy as before, so you can unlock them by running `fscrypt unlock` on any directory uses the same encryption policy, such as the one that they were moved out of originally.
